pacai/bin/capture.py

```diff
@@ -602,8 +600,13 @@ def readCommand(argv):
     args['redTeamName'] = options.red
     args['blueTeamName'] = options.blue
 
-    if options.fixRandomSeed:
-        random.seed(FIXED_SEED)
+    # If no seed entry generate a random seed value.
+    seed = options.seed
+    if seed is not None:
+        random.seed(options.seed)
+    else:
+        seed = random.randint(0, 2**32)
```

Comment:
You never seed in this branch (you generate a seed, but never use it).
The structure should look like:
```
seed = options.seed
if (seed is None):
   seed = random.randint(0, 2**32)
random.seed(seed)
logging.debug('Seed value: ' + str(seed))
```